src/angular/input/input.spec.ts

```diff
@@ -425,53 +427,6 @@ describe('SbbInput without forms', () => {
     expect(container.classList).not.toContain('sbb-focused');
   }));
 
-  it('should not highlight when focusing a readonly input', fakeAsync(() => {
```

Comment:
Why are they not needed anymore?

Reply:
This is based on a change in components. They originally mimicked browser behavior, but this behavior was changed in browsers.